digitalforms-api/src/main/java/ca/bc/gov/open/digitalformsapi/viirp/model/vips/CreateImpoundmentServiceResponse.java

```diff
@@ -0,0 +1,42 @@
+package ca.bc.gov.open.digitalformsapi.viirp.model.vips;
```

Comment:
Because it specific to the VIPS API, I wanted to keep it separate from the VIIRP API and hence why I've created the VIPS model package. Gets too confusing if everything is in the same 'model' pkg.